src/client/automation/playback/move.js

```diff
@@ -97,7 +98,11 @@ export default class MoveAutomation {
         var moveOptions = new MoveOptions({
             modifiers: e.message.modifiers,
             offsetX:   intersectionRelatedToIframe.x + iframeBorders.left + iframePadding.left,
-            offsetY:   intersectionRelatedToIframe.y + iframeBorders.top + iframePadding.top
+            offsetY:   intersectionRelatedToIframe.y + iframeBorders.top + iframePadding.top,
+
+            // NOTE: we should not run scrolling because it was scrolled to the
+            // target element in the active window before the request (GH-847)
```

Comment:
```
            // NOTE: we should not perform scrolling because the active window was already
            // scrolled to the target element before the request (GH-847)
```